tests/GenericSparqlTest.php

```diff
@@ -178,6 +178,133 @@ public function testQueryConceptsAlphabeticalLimitAndOffset() {
     $this->assertEquals($expected, $actual);
   }
 
+  /**
+   * @covers GenericSparql::queryConceptsAlphabetical
+   * @covers GenericSparql::generateAlphabeticalListQuery
+   * @covers GenericSparql::transformAlphabeticalListResults
+   */
+  public function testQualifiedNotationAlphabeticalList() {
+    $voc = $this->model->getVocabulary('test-qualified-notation');
+    $res = new EasyRdf\Resource("http://www.w3.org/2004/02/skos/core#notation");
+    $sparql = new GenericSparql('http://localhost:13030/skosmos-test/sparql', $voc->getGraph(), $this->model);
+
+    $actual = $sparql->queryConceptsAlphabetical("a", "en", $limit = null, $offset = null, $classes=null, $showDeprecated=false, $qualifier=$res);
```

Comment:
Here (and in a few similar places) you use "named parameters" (like in Python) but PHP doesn't support it, so in effect you are declaring a bunch of unused local variables. SonarCloud complains about them.